docs/config/dfget_config_template.yml

```diff
@@ -14,11 +14,15 @@ localLimit: 20M
 
 # Minimal rate about a single download task, format: G(B)/g/M(B)/m/K(B)/k/B
 # pure number will also be parsed as Byte.
-minRate: 512
+minRate: 512B
```

Comment:
set the value as same as the value in the code

Reply:
DONE.